### DISCUSSION TITLE
是否有在Mac和window环境下DTK相关库的编译教程

### DISCUSSION THREAD

**Discussion by tubufeng** [linuxdeepin/developer-center#3329] [Q&A | 问答板块]
如题，感谢。

**Comment by BLumia** [comment]
v20 开始的早期 DTK 版本应该还是能在 mac 和 Windows 下构建的，现在的应该是不能的，需要一些移植工作。如果你打算自己构建试试看的话，如果你是 Windows 用户，我个人比较建议你使用 MSYS2 或 KDE Craft 环境，然后依次构建 dtkcommon，dtkgui，dtkcore，dtkwidget（顺序不保证没反）。肯定会碰到很多平台移植问题，如果有兴趣的话可以发 PR :)

作为补充，我们这边是打算在 v23 发布前，DTK 项目本身切换到 CMake 后再重新进行 mac 与 Windows 的支持。
